src/basic/string-util.h

```diff
@@ -129,6 +129,9 @@ static inline bool _pure_ in_charset(const char *s, const char* charset) {
         return s[strspn(s, charset)] == '\0';
 }
 
+static inline bool char_is_cc(char p) {
+        return (p >= 0 && p < ' ') || p == 127;
```

Comment:
hmm, on archs that have unsigned chars the first check might result in a warning as being redundant... maybe cast to unsigned here explicitly...

(but also, if the CI is happy I don't care)

Reply:
I don't have access to a BE machine. But I think gcc/clang are smart enough to know that `char` is special…